wiki/Beatmap/Star_rating/ja.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - SR
+  - stars
+---
+
+# 難易度
+
+**難易度** (Star rating, SR)は、osu!によって自動的に計算されるビートマップの難易度を表します。難易度は、パフォーマンスポイントに大きな影響を与えます。難易度は曲の選択画面とビートマップ情報ページで確認できます。
```

Comment:
many links missing

for ref:

`**Star rating** (***SR***) describes a [beatmap](/wiki/Beatmap)'s difficulty, which is automatically calculated by the osu! client. Star rating plays a large role in how [performance points](/wiki/Performance_points) are awarded. This number can be seen on the [song selection screen](/wiki/Client/Interface#song-select) and on a beatmap's info page.`